[Bug] [DolphinScheduler : 3.2.2]K8s Task Plugin Fails with NullPointerException After Refactor - Namespace Selection Missing

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

Description:
After the recent K8s task plugin refactoring, we're experiencing a critical issue where K8s tasks fail with a NullPointerException. The error occurs in the buildCommand method of K8sTask.java (line 86) when trying to execute or cancel a task.

Error Details:

Primary error occurs in K8sTask.buildCommand()

Secondary error occurs during task cancellation

The root cause appears to be related to missing namespace selection functionality after the refactor

Steps to Reproduce:

Create a K8s task in DolphinScheduler

Attempt to execute the task

Observe NullPointerException in logs

Expected Behavior:

Users should be able to select/configure a Kubernetes namespace for the task

Tasks should execute properly with proper namespace configuration

Task cancellation should work without exceptions

Actual Behavior:

Task execution fails with NullPointerException

Task cancellation also fails with same exception

No apparent way to configure namespace in UI

Environment:

DolphinScheduler version: 3.2.2
Kubernetes version: v1.23.17

Additional Context:
The stack trace shows the error propagates from:
```
[ERROR] 2025-05-15 11:18:18.375 +0800 - k8s task submit failed with error
[ERROR] 2025-05-15 11:18:18.375 +0800 - Task execute failed, due to meet an exception
org.apache.dolphinscheduler.plugin.task.api.TaskException: Execute k8s task error
	at org.apache.dolphinscheduler.plugin.task.api.k8s.AbstractK8sTask.handle(AbstractK8sTask.java:59)
	at org.apache.dolphinscheduler.server.worker.runner.DefaultWorkerTaskExecutor.executeTask(DefaultWorkerTaskExecutor.java:51)
	at org.apache.dolphinscheduler.server.worker.runner.WorkerTaskExecutor.run(WorkerTaskExecutor.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.NullPointerException: null
	at org.apache.dolphinscheduler.plugin.task.k8s.K8sTask.buildCommand(K8sTask.java:86)
	at org.apache.dolphinscheduler.plugin.task.api.k8s.AbstractK8sTask.handle(AbstractK8sTask.java:52)
	... 5 common frames omitted
[ERROR] 2025-05-15 11:18:18.375 +0800 - Cancel task failed, this will not affect the taskInstance status, but you need to check manual
java.lang.NullPointerException: null
	at org.apache.dolphinscheduler.plugin.task.k8s.K8sTask.buildCommand(K8sTask.java:86)
	at org.apache.dolphinscheduler.plugin.task.api.k8s.AbstractK8sTask.cancelApplication(AbstractK8sTask.java:83)
	at org.apache.dolphinscheduler.plugin.task.api.AbstractRemoteTask.cancel(AbstractRemoteTask.java:39)
	at org.apache.dolphinscheduler.server.worker.runner.WorkerTaskExecutor.cancelTask(WorkerTaskExecutor.java:133)
	at org.apache.dolphinscheduler.server.worker.runner.WorkerTaskExecutor.afterThrowing(WorkerTaskExecutor.java:114)
	at org.apache.dolphinscheduler.server.worker.runner.DefaultWorkerTaskExecutor.afterThrowing(DefaultWorkerTaskExecutor.java:61)
	at org.apache.dolphinscheduler.server.worker.runner.WorkerTaskExecutor.run(WorkerTaskExecutor.java:179)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
[INFO] 2025-05-15 11:18:18.378 +0800 - Get a exception when execute the task, will send the task status: FAILURE to master: dolphinscheduler-worker-1.dolphinscheduler-worker-headless:1234
[INFO] 2025-05-15 11:18:18.378 +0800 - FINALIZE_SESSION
```
This appears to be a regression introduced by the recent plugin refactoring, where namespace handling was either removed or not properly implemented.

Suggested Priority:
High - This completely breaks K8s task functionality



### What you expected to happen

![Image](https://github.com/user-attachments/assets/ec8bf3e6-8c2b-4a64-836c-12d5d1253083)

![Image](https://github.com/user-attachments/assets/17b19c35-82fc-4de6-9eb6-360aba2f700c)


### How to reproduce


version: 3.2.0

![Image](https://github.com/user-attachments/assets/c3109709-a466-498b-be53-0a19376cbdfd)

version:3.2.2

![Image](https://github.com/user-attachments/assets/14d315f1-8b78-40e5-9046-f4ac5a1d5fb8)

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-k8s.ts
+++ b/dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-k8s.ts
@@ -14,7 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { useCustomParams, useCustomLabels, useNodeSelectors } from '.'
+import {
+  useCustomParams,
+  useNamespace,
+  useCustomLabels,
+  useNodeSelectors
+} from '.'
 import type { IJsonItem } from '../types'
 import { useI18n } from 'vue-i18n'
 
@@ -22,6 +27,7 @@ export function useK8s(model: { [field: string]: any }): IJsonItem[] {
   const { t } = useI18n()
 
   return [
+    useNamespace(),
     {
       type: 'input-number',
       field: 'minCpuCores',
@@ -107,11 +113,7 @@ export function useK8s(model: { [field: string]: any }): IJsonItem[] {
       field: 'nodeSelectors',
       name: 'node_selectors'
     }),
-    ...useCustomParams({
-      model,
-      field: 'localParams',
-      isSimple: model.readonly
-    })
+    ...useCustomParams({ model, field: 'localParams', isSimple: false })
   ]
 }
 

